app/controllers/application_controller.rb

```diff
@@ -107,32 +106,6 @@ def create_sso_account(user, personish, timeout, account_role = "individual")
 
   private
 
-  def check_concurrent_sessions
-    return unless preferred_user_access(current_user)
-
-    flash[:error] = l10n('devise.sessions.signed_out_concurrent_session')
-    sign_out current_user
-  end
-
-  # preferred_user_access feature should be turned on DC & disabled in ME
-  def preferred_user_access(current_user)
```

Comment:
@TreyE The problem with this is I think DC wants to limit admin concurrent sessions only and not for all the users.
@jacobkagon @jayreddy519 Should be able to provide more information about the DC requirement.